pages/common/clifm.md

```diff
@@ -0,0 +1,36 @@
+# clifm
+
+> The command line file manager.
+> More information: <https://github.com/leo-arch/clifm>.
+
+- Change to/open the file whose ELN (entry list number) is 12 (ELN can be replaced by the actual file name, e.g. /etc):
```

Comment:
These examples are great, but how does one start the program in the first place? tldr pages should tell a story of how the command is used, with the first example on the page being the command users will generally want to use first.

Also, this specific example may be best served as 2 separate examples rather than being rolled into 1.

Reply:
Hi @sbrl. Thanks for your kind review. 

To start the program one just runs `clifm`, which will open a new command line to run clifm specific commands. I just thought that, having a limited amount of available examples (8), it would be better to use them for less intuitive commands. The same goes for the idea of splitting the example into two. However, I perfectly understand the idea of a commands story. It makes sense in the context of `tldr`. I'll rewrite the examples and push again.

Reply:
Remember that tldr pages are for both beginners and experts. Space is limited, so we have to balance the need for a page to be useful to both beginners and experts. To this end, larger numbers of advanced examples may be better suited to a dedicated cheatsheet in a different project?

Reply:
Hi @sbrl. I understand your point. Because CliFM is a command line program, it has a lot of available commands. This is why I designed a well documented manpage and a quite big Wiki. However, I really value your work with `tldr`, and I think clifm should be here: a little introduction is always a good thing. I making short intra-program help sections myself: having only a huge manpage/Wiki is far from optimal.